kustomize/commands/internal/util/util.go

```diff
@@ -96,3 +97,24 @@ func trimQuotes(s string) string {
 	}
 	return s
 }
+
+// Checks whether the given glob pattern intentionally wants to include hidden files
+func ShouldGlobHiddenFiles(pattern string) bool {
+	return strings.HasPrefix(filepath.Base(pattern), ".")
+}
+
+// Removes paths containing hidden files/folders from a list of paths
+func RemoveHiddenFiles(paths []string) []string {
+	if len(paths) == 0 {
+		return paths
+	}
+	var result []string
+	for _, path := range paths {
+		if strings.HasPrefix(filepath.Base(path), ".") {
+			continue
+		} else {
+			result = append(result, path)
```

Comment:
Let's invert this if statement and use the function you've written above:

```
		if !IsHiddenFilePath(path){
			result = append(result, path)
		}	
```

No `else` needed